engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -163,6 +163,17 @@ FLUTTER_DARWIN_EXPORT
       allowHeadlessExecution:(BOOL)allowHeadlessExecution
           restorationEnabled:(BOOL)restorationEnabled NS_DESIGNATED_INITIALIZER;
 
+/**
+ * Returns engine for the identifier. The identifier must be valid for an engine
+ * that is currently running, otherwise the behavior is undefined.
+ *
+ * The identifier can be obtained in Dart code through
+ * `PlatformDispatcher.instance.engineId`.
+ *
+ * This function must be called on the main thread.
+ */
++ (nullable FlutterEngine*)engineForId:(int64_t)identifier;
```

Comment:
Nit for either a private version, or for when this is added publicly: `engineForIdentifier:`

Reply:
Done.